android/app/src/main/res/values-es-rPR/strings.xml

```diff
@@ -2,9 +2,6 @@
 <resources>
   <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
   <string name="notification_channel_description">Notificaciones de PathCheck BT</string>
-  <!-- Server URI for diagnosis key file download -->
-  <string name="key_server_download_base_uri">https://federationout-2kabcv6c4a-uc.a.run.app/</string>
```

Comment:
this is flagged as a non-translatable key so no other file has this "translated". but this is the only file in which there was a value in it, which is different from our standard value for this key "https://storage.googleapis.com/exposure-notification-export-svhfv/". do we want to add this key/ value back into the rPR file @aledustet ?